.github/workflows/ci.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
       - uses: microsoft/playwright-github-action@v1
       - name: Install dependencies and run tests
         run: |
-          cd release-packages && node code-server*-linux-amd64 &
+          cd release-packages && node PASSWORD=e45432jklfdsab code-server*-linux-amd64 &
```

Comment:
Open to alternatives if this isn't the best approach. I considered reading from `config.yaml` but wasn't sure if that would be difficult to read since I think the path is different locally vs ci.

Reply:
I think this is perfect.

Reply:
Going with the "set `PASSWORD` as an env var in `ci` approach so removing this